### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by gretzke**
This PR includes deployment details for the Emission Manager upgrade on the Goerli testnet

[Default Emission Manager](https://goerli.etherscan.io/address/0x0e481cfeB712d37A0C6cd4e2801cA01E09Bc1A9d) ([Implementation](https://goerli.etherscan.io/address/0x16258ebEa8eb68271Ca2677aBE05256265398A4D))

<details>
<summary>Inputs</summary>
<table>
    <tr>
        <th>Parameter</th>
        <th>Value</th>
    </tr>
    
<tr>
    <td>migrationProxy</td>
    <td>0x5c5589fca76237Ed00BA024e19b6C077a108f687</td>
</tr>

<tr>
    <td>stakeManager</td>
    <td>0x00200eA4Ee292E253E6Ca07dBA5EdC07c8Aa37A3</td>
</tr>

<tr>
    <td>treasury</td>
    <td>0x531c7Befe78B6496e5753815ab3d3Cc024c1E842</td>
</tr>
</table>
</details>

### CODE REVIEW

**Review by DhairyaSethi — APPROVED** [review]

File: src/PolygonEcosystemToken.sol [hunk 1]
```diff
@@ -15,7 +15,7 @@ contract PolygonEcosystemToken is ERC20Permit, AccessControlEnumerable, IPolygon
     bytes32 public constant CAP_MANAGER_ROLE = keccak256("CAP_MANAGER_ROLE");
     bytes32 public constant PERMIT2_REVOKER_ROLE = keccak256("PERMIT2_REVOKER_ROLE");
     address public constant PERMIT2 = 0x000000000022D473030F116dDEE9F6B43aC78BA3;
-    uint256 public mintPerSecondCap = 10e18; // 10 POL tokens per second
+    uint256 public mintPerSecondCap = 13.37e18;
```
Comment (web3security) [inline src/PolygonEcosystemToken.sol:18]: As discussed, I would suggest to add a few more tests to check which are the new emission boundaries or limits today. 

I have just made some fuzzing around this. As a conclusion, it seems that in case of frequent minting the system will reach the limit in less than 12 years (from 23 years) but there might be edge situations where we can mint a bigger supply just waiting a bunch of years without minting any token as max emission depends on last minting timestamp.

Summarizing, there were some important behavioral changes on the emission system and its limits by this small change, so I would suggest we run some few more simulations as well as fuzzing cases next time if possible.  Thanks!

**Review by web3security — APPROVED** [review]
LGTM. Great job! 🎉

**Review by dev1644 — APPROVED** [review]
LGTM!

**Comment by gretzke** [comment]
This PR now also includes the deployment details for the POL Mainnet deployment


- [Polygon Migration](https://etherscan.io/address/0x29e7DF7b6A1B2b07b731457f499E1696c60E2C4e) ([Implementation](https://etherscan.io/address/0x550B7CDaC6F5a0d9e840505c3Df74aC045530446))
- [Default Emission Manager](https://etherscan.io/address/0xbC9f74b3b14f460a6c47dCdDFd17411cBc7b6c53) ([Implementation](https://etherscan.io/address/0x2126E6952C3af75C9D4CF21f63F509195C79ce44))
- [Polygon Ecosystem Token](https://etherscan.io/address/0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6)

<details>
<summary>Inputs</summary>
<table>
    <tr>
        <th>Parameter</th>
        <th>Value</th>
    </tr>
    
<tr>
    <td>matic</td>
    <td>0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0</td>
</tr>

<tr>
    <td>protocolCouncil</td>
    <td>0x29A6f32f36EDeD399763524018F17F03B1435b18</td>
</tr>

<tr>
    <td>treasury</td>
    <td>0x2ff25495d77f380d5F65B95F103181aE8b1cf898</td>
</tr>

<tr>
    <td>stakeManager</td>
    <td>0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908</td>
</tr>

<tr>
    <td>emergencyCouncil</td>
    <td>0x37D085ca4a24f6b29214204E8A8666f12cf19516</td>
</tr>
</table>
</details>
